webapp/challenge/admin/challenge.py

```diff
@@ -1,11 +1,23 @@
-from challenge.models import Challenge
 from django.contrib import admin
 
+from challenge.models import Challenge
+from file_manager.models import Image
+
 
 class ImageInline(admin.TabularInline):
     model = Challenge.images.through
 
 
+def copy_challenge(modeladmin, request, queryset):
+    for challenge in queryset:
+        if challenge.thumbnail:
+            temp_image = Image(file=challenge.thumbnail.file, uploader=challenge.thumbnail.uploader)
+            temp_image.save()
+            challenge.thumbnail = temp_image
+        challenge.pk = None
+        challenge.save()
```

Comment:
```suggestion
    for challenge in queryset:
        challenge.deep_copy()
```

challenge 모델에 deep_copy 메서드를 하나 두고, 관리를 하면 좋을 것 같습니다.

Reply:
그럼 이 부분은 위와 같이 되겠죠?

Reply:
```python

def deep_copy(self):
    new_image = Image() # 생략
    new_challenge = Challenge(
        title=self.title,
        thumbnail=new_image,
        ## 생략
    )

```

Reply:
@Jin409  challenge admin에서가 아닌 모델에서를 말한거였어요

Reply:
아핫 네 수정할게요!